dt-core/admin/menu/tabs/tab-custom-fields.php

```diff
@@ -881,6 +882,8 @@ private function add_field( $post_type ){
         global $wp_post_types;
         $post_type = sanitize_text_field( wp_unslash( $post_type ) );
         $tile_options = DT_Posts::get_post_tiles( $post_type );
+        global $wp_post_types;
```

Comment:
`global $wp_post_types`  is already declared a few lines above

Reply:
Can't have that!

Reply:
Thanks for spotting it.